tests/test_fields.py

```diff
@@ -1305,3 +1307,37 @@ def test_get_col(self):
         ).get()
 
         self.assertIsInstance(instance, RelatedDateTime)
+
+    def test_write_to_diff_keys(self):
```

Comment:
How can we be certain (assert) that this is using different settings? - I would expect these tests to have passed without the settings changes.

Reply:
The only thing I can think of is to examine the actual sql sent to the db possibly using `django.db`
```
from django.db import connection

self.assertEqual(connection.queries[0], something)
 ```

Alternatively using `unittest.mock`

Reply:
I agree - I did figure out how to do it. You have to explicitly set DEBUG = True *inside* the test in order to log the queries run. Resolved.